solution/server/trace-pkg-config.yaml

```diff
@@ -0,0 +1,52 @@
+# Global options
```

Comment:
Can you help me understand why you chose this library?

Reply:
Talking to Pedro when he introduced me to Superformula, I was really interested to know what was the company's stack, and he gave me a little tour of the techs used and I was amazed by "trace-pkg" because it not only creates the package to be deployed, but it does it in an optimized way. It is also very simple to configure and very easy to integrate with Terraform. I could not find a better alternative to it.